analysis/dem_plots_arina.R

```diff
@@ -0,0 +1,32 @@
+# Set up
+library(tidyverse)
+library(lubridate)
+df <- read.csv("output/dem/dem_qof_measures.csv")
+
+# summary(df) - confirms that there are 37 missing ratios and 24 missing imds
+
+# Dementia prevalence plot, stratified by IMD
+df %>%
+filter(!is.na(imd)) %>%
+ggplot(aes (x= ymd(interval_start), y= ratio, color = as.factor(imd))) +
+geom_line()+
+labs (title = "Dementia prevalence over time, per IMD group", 
+x = "", 
+y ="Prevalence of dementia",
+color= "Index of multiple depravation") +
+theme_light()
```

Comment:
similar to all the Python style guides we spoke about, there are similar conventions for formatting R code, for example see https://style.tidyverse.org/ggplot2.html for the ggplot2 section. Also, if you do not want to visualise any labels, it's usually better to use `NULL` instead of an empty string `""`:

```suggestion
df %>%
  filter(!is.na(imd)) %>%
  ggplot(aes (x = ymd(interval_start), y = ratio, color = as.factor(imd))) +
  geom_line()+
  labs (
    title = "Dementia prevalence over time, per IMD group", 
    x = NULL, 
    y = "Prevalence of dementia",
    color = "Index of multiple depravation"
    ) +
  theme_light()
```